src/options.js

```diff
@@ -13,4 +15,21 @@ const options = {
 	_catchError
 };
 
+Object.defineProperty(options, 'vnode', {
+	get() {
+		return vnodeHook;
+	},
+	set(newHook) {
+		if (newHook) {
+			let oldHook = vnodeHook;
+			vnodeHook = (vnode) => {
+				newHook(vnode);
+				if (oldHook) oldHook(vnode);
```

Comment:
Yes!

Though I think we can't auto apply the old hook cuz I think some implementations need to invoke their logic before and/or after the previous hooks. For example, if I want to modify something `preact/hooks` does I may want to invoke my implementation after invoking the previous hook to ensure hooks stuff runs first.

However, I like this idea of if simplifying hook integrations. I think we had previously discussed adding something like a `preact/shared-internals` package or something and exposing a [hook helper like we have in signals](https://github.com/preactjs/signals/blob/6b6af05d433b4d2a8da0be5a65ba5305d49e7b73/packages/preact/src/index.ts#L27-L31) that would always ensure the first argument to the hook is the previous hook. That doesn't need to happen in this PR but just thought I'd write it down cuz I'm not sure we have that written down elsewhere 😅

Reply:
I updated it, just have one test that seems to be failing, not entirely sure how to remedy it yet will look deeper over the next week.

Reply:
The perf diff does not seem to be working atm